week2/homework/src/methods.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+const theMethods = {
+  // ADD:
+  add: item => {
+    readFile(__dirname + '/todo.txt', 'utf-8').then(content => {
+      if (content === '') {
+        fs.writeFile(__dirname + '/todo.txt', item, err => {
+          console.log("an item 's been added to the toDo list...");
+        });
+      } else {
+        fs.appendFile(__dirname + '/todo.txt', `\n${item}`, err => {
+          if (err) throw err;
+          console.log("an item 's been added to the toDo list...");
+        });
+      }
+    });
+  },
+
+  // DELETE:
+  delete: id => {
+    readFile(__dirname + '/todo.txt', 'utf-8').then(content => {
+      const contentArr = content.split('\n');
+      contentArr.forEach((item, index) => {
+        if (id == index + 1) {
+          const removed = contentArr.splice(id - 1, 1);
```

Comment:
`removed` is never used. Not an "error", but it's always good to remember to remove unused code at the end.
It makes the code neater, more elegant and readable 👍

Reply:
done.